src/main/cluster.ts

```diff
@@ -373,10 +368,12 @@ export class Cluster implements ClusterModel, ClusterState {
   /**
    * @internal
    */
-  protected async ensureKubectl() {
-    this.kubeCtl = new Kubectl(this.version);
+  async ensureKubectl() {
+    this.#kubeCtl ??= new Kubectl(this.version);
```

Comment:
Was there a bug before? 🤔

Reply:
I think this is more of an optimization since `Kubectl` merely represents an on-disk binary. Though I did run into several crashes where `cluster.kubeCtl` was `undefined` while doing this.